llvm/lib/Analysis/UniformityAnalysis.cpp

```diff
@@ -88,19 +129,6 @@ void llvm::GenericUniformityAnalysisImpl<
   }
 }
 
-template <>
-bool llvm::GenericUniformityAnalysisImpl<SSAContext>::isDivergentUse(
```

Comment:
Why was this moved? If there is a change in the body, can you first move the function as a separate NFC commit, so that the diff is more useful?